recipes-test/packetdrill/packetdrill_20210310.bb

```diff
@@ -0,0 +1,31 @@
+SUMMARY = "Packetdrill: Network stack testing tool"
+DESCRIPTION = "The packetdrill scripting tool enables quick, precise tests for \
+entire TCP/UDP/IPv4/IPv6 network stacks, from the system call layer down to \
+the NIC hardware."
+HOMEPAGE = "https://github.com/google/packetdrill.git"
+
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/GPL-2.0;md5=801f80980d171dd6425610833a22dbe6"
+
+SRC_URI = "git://github.com/google/packetdrill.git;protocol=https"
+
+SRCREV = "75fe342a88a74bb610a921a3090b3611a2624ff4"
+
+S = "${WORKDIR}/git/gtests/net/packetdrill/"
+
+DEPENDS = "bison-native flex-native rsync-native"
+RDEPENDS_${PN} = "python3"
+
+inherit autotools-brokensep
+
+do_install() {
+    install -D -m 0755 -t ${D}/opt/packetdrill/ \
+        checksum_test \
+        packet_parser_test \
+        packet_to_string_test \
+        packetdrill
```

Comment:
we need to install dir common and tcp
also we need to install packetdrill/run_all.py

Reply:
Noted! Thanks for the review.